modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -80,6 +82,22 @@ Failed to bind volumes: provisioning failed for PVC
 ----
 +
 You must use the NFS storage class. Cinder storage is not supported. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[(*BZ#1857784*)]
+
+ifeval::["{HCOVersion}" == "2.4.1"]
```

Comment:
virt-document-attributes currently has this set to 2.4.0 so this isn't included in the current build. 
Is this attribute going to be updated in a different PR closer to release?

Reply:
I certainly hope so. 2.4.1 is supposed to be released tomorrow. Isn't someone on your team responsible for updating the attributes with each release?